worldedit-core/src/main/java/com/fastasyncworldedit/core/queue/IBatchProcessor.java

```diff
@@ -67,16 +68,29 @@ default boolean trimY(IChunkSet set, int minY, int maxY, final boolean keepInsid
             for (int layer = set.getMinSectionPosition(); layer <= minLayer; layer++) {
                 if (set.hasSection(layer)) {
                     if (layer == minLayer) {
-                        char[] arr = set.loadIfPresent(layer);
-                        if (arr != null) {
-                            int index = (minY & 15) << 8;
-                            for (int i = 0; i < index; i++) {
-                                arr[i] = 0;
+                        if(!(set instanceof IntSetBlocks)) {
```

Comment:
For all these checking char vs int, it would be slightly more readable to have int in the if block, or check for Char[Set|Get]Block instance so you're not checking negation every time.

Reply:
I went through and inverted the ```if``` checks so it should be a lot more readable now.